### PR TITLE
Add a new line feed to the end of an output format

### PR DESCRIPTION
**Pull request by Luni-4**
This PR adds a new line feed to the end of an output format

### CODE REVIEW

**Comment by Luni-4** [comment]
@marco-c 

Can we merge this one?

**Comment by marco-c** [comment]
Why do you want to add a newline at the end of the output?

**Comment by Luni-4** [comment]
In this way distinct files are printed on different lines and it is much easier to retrieve them

File: rust-code-analysis-cli/src/formats.rs [hunk 1]
```diff
@@ -42,17 +42,17 @@ impl Format {
                     } else {
                         serde_json::to_string(&space).unwrap()
                     };
-                    write!(stdout, "{}", json_data)
+                    write!(stdout, "{}\n", json_data)
```
Comment (marco-c) [inline rust-code-analysis-cli/src/formats.rs:45]: You can use `writeln!` instead

**Review by marco-c — APPROVED** [review]
There is a compilation error, but it is not your fault.

**Pull request merged**